lntest/itest/lnd_test.go

```diff
@@ -4671,6 +4671,37 @@ func testMultiHopPayments(net *lntest.NetworkHarness, t *harnessTest) {
 		lnd.DefaultBitcoinTimeLockDelta, maxHtlc, carol,
 	)
 
+	// Before we start sending payments, subscribe to htlc events for each
```

Comment:
We are in the process of splitting lnd itests into their own files when we touch a file. You could also do that for this PR (non blocking).